public/app/plugins/panel/nodeGraph/utils.ts

```diff
@@ -45,6 +45,8 @@ export function shortenLine(line: Line, sourceNodeRadius: number, targetNodeRadi
 }
 
 export type NodeFields = {
+  fixed_x?: Field;
+  fixed_y?: Field;
```

Comment:
Why the underscore naming here? I think we should also just keep the same camelCase naming